pygmt/src/select.py

```diff
@@ -147,6 +147,16 @@ def select(data=None, outfile=None, **kwargs):
         - :class:`pandas.DataFrame` table if ``outfile`` is not set.
         - None if ``outfile`` is set (filtered output will be stored in file
           set by ``outfile``).
+
+    Example
+    -------
+    >>> import pygmt  # doctest: +SKIP
+    >>> # Load a table of ship observations of bathymetry off Baja California
+    >>> data = pygmt.datasets.load_sample_data(name="bathymetry")   # doctest: +SKIP
+    >>> # Only return the data points that lie within the region between
+    >>> # longitudes 246 and 247 and latitudes 20 and 21
+    >>> pygmt.select(data=ship_data, region=[246,247,20,21])
+    ... )  # doctest: +SKIP
```

Comment:
```suggestion
    >>> pygmt.select(data=ship_data, region=[246, 247, 20, 21])
```

Reply:
When `# doctest: +SKIP` was left out of later lines of code for the `blockm*` examples, it raised an error during testing because pygmt hadn't been imported. I think it is needed in subsequent lines.

Reply:
> When `# doctest: +SKIP` was left out of later lines of code for the `blockm*` examples, it raised an error during testing because pygmt hadn't been imported. I think it is needed in subsequent lines.

OK. I've updated my suggestion.